extensions/amp-story-subscriptions/0.1/amp-story-subscriptions.css

```diff
@@ -66,6 +68,39 @@ amp-story:not([supports-landscape]) + amp-subscriptions-dialog.i-amphtml-story-s
 .i-amphtml-story-subscriptions-dialog-banner {
   height: 2.54em !important;
   background: #EDEDED !important;
+  display: flex !important;
+  justify-content: flex-end !important;
+  align-items: center !important;
+  padding: 0 0.3em !important;
+}
+
+.i-amphtml-story-subscriptions-dialog-banner-button {
+  display: flex !important;
+  justify-content: flex-end !important;
+  align-items: center !important;
+  letter-spacing: 0.015em !important;
+  font-size: 0.75em !important;
+  opacity: 0 !important;
+  transition: opacity 0.3s cubic-bezier(0.42, 0.0, 1, 1) !important;
+}
+
+/* TODO: confirm with UX about better animation. */
+.i-amphtml-story-subscriptions-dialog-banner-button.i-amphtml-story-subscriptions-dialog-banner-button-visible {
+  opacity: 1 !important;
+}
+
+.i-amphtml-story-subscriptions-dialog-banner-button:after, [dir="rtl"] .i-amphtml-story-subscriptions-dialog-banner-button:after {
```

Comment:
The `[dir="rtl"] .i-amphtml-story-subscriptions-dialog-banner-button:after` selector isn't needed here.
The styles will still apply in `rtl`.